projects/spring-security/project.yaml

```diff
@@ -13,4 +13,5 @@ vendor_ccs:
   - "yakdan@code-intelligence.com"
   - "glendowne@code-intelligence.com"
   - "patrice.salathe@code-intelligence.com"
-  - "hlin@code-intelligence.com"
\ No newline at end of file
+  - "hlin@code-intelligence.com"
+  - "yoshi.weber@gmail.com"
```

Comment:
Are you a project maintainer? Maybe add yourself to auto_ccs: instead please?

Reply:
He's not. He just wants to get access to the platform and the bug reports